test/app/submissions_test.rb

```diff
@@ -43,7 +43,7 @@ def assert_response_status(expected_status)
   end
 
   def test_guests_can_view_submissions
-    Attempt.new(alice, Iteration.new('word-count/file.rb' => 'CODE')).save
+    Attempt.new(alice, Iteration.new('ruby/file.rb' => 'CODE')).save
```

Comment:
The `word-count` slug wouldn't work because it didn't exist inside the tracks fixture. I decided to change this to `ruby` instead. What I found out though was an underlying error (or maybe not). When I use the `X::Track.find` call with an invalid slug, it fails. I guess for this test I could also stub out `X::Language.of` to handle the `word-count` slug.